MatchServer/MMatchServer.cpp

```diff
@@ -562,6 +568,7 @@ void MMatchServer::Destroy()
 	m_AsyncProxy.Destroy();
 	MGetMatchShop()->Destroy();
 	m_SafeUDP.Destroy();
+	MGetGunGame()->~GunGame();
```

Comment:
Gotcha. In the destructor, am I clearing the map correctly? Never cleared a map that contained a vector of pointers. I assume it's correct.